tools/nxfuse/README_SMARTFS.md

```diff
@@ -1,20 +1,86 @@
-nxfuse
-======
+# How to make smartfs user image for TizenRT
+
+## Contents
+> [How to Enable SmartFS user image](#how-to-enable-smartfs-user-image)  
+> [About NxFuse tool](#about-nxfuse-tool)  
```

Comment:
I have no idea that means we should follow or not. That's why I suggest a title as APPENDIX.

Reply:
@sunghan-chang: Can you please explain more on what todo? Review comment not clear to me